grpc/response.go

```diff
@@ -0,0 +1,32 @@
+package grpc
+
+import (
+	"strings"
+
+	"github.com/gojek/fiber"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/grpc/status"
+)
+
+type Response struct {
```

Comment:
Can you mantain consistency between `grpc.Request` and `grpc.Response` ? In this case I think it's better to have something like
```
type Response struct {
  Metadata metadata.MD
  Payload proto.Message
  Status status.Status
}
```

Reply:
yep, from the above we can change the payload back to `proto.Message`. I dont think there is any usage for status for `Request`, we can stick to what u suggested, minus status for `Request`.

```
type Request struct {
  Metadata metadata.MD
  Payload proto.Message
}
```
```
type Response struct {
  Metadata metadata.MD
  Payload proto.Message
  Status status.Status
}
```

Reply:
If we remove `Status` how does the client know what causes an error?